src/components/side-panel/panel-header.js

```diff
@@ -123,17 +122,10 @@ const StyledPanelDropdown = styled.div`
 export const PanelAction = ({item, onClick}) => (
   <StyledPanelAction className="side-panel__panel-header__action"
     data-tip data-for={`${item.id}-action`} onClick={onClick}>
+    <p style={{fontWeight:'bold'}}>Share</p>
```

Comment:
space after colon

Reply:
@akre54 Fix this, thanks for your review